spec/build_spec_helper.cr

```diff
@@ -0,0 +1,77 @@
+require "./spec_helper"
+require "./support/helpers/cli_helper"
+
+include CLIHelper
+
+macro generate_app(*options)
```

Comment:
why this has to be a macro?

Reply:
@eliasjpr I did this to preserve every spec scope (actually this just avoid boilerplate code :sweat_smile: )

Reply:
I think this can be done without a macro, I will give it a try. The specs should run independently

Reply:
I removed macros :+1:

Also the `describe` are running their specs `it` independently :tada: